flytekit/core/notification.py

```diff
@@ -1,3 +1,19 @@
+"""
+Notifications are primarily used when defining Launch Plans (also can be used when launching executions) and will trigger
+the Flyte platform to send emails when a workflow run reaches certain stages (fails or suceeds, etc.).
+
+.. note::
+
+    Notifications require some setup and configuration on the Flyte platform side. Please contact your Flyte platform
+    admins to get this feature enabled. See :std:doc:`flyte:howto/notifications`
+
+Each notification type takes a list of :py:class:`flytekit.models.core.execution.WorkflowExecutionPhase` and a list of
+emails. Even though there are different notification classes in this module, they all just send email. The differentiation
+offers semantic meaning to the end-user but do not functionally behave differently.
```

Comment:
Can you add that for successful integration with slack and pagerduty those email APIs must have been set-up correctly?